objects/pt-BR.json

```
@@ -1,19 +1,19 @@
 {
   "rct2.glthent": {
     "reference-name": "'Goliath' Sign",
-    "name": "Placa 'Golias'"
+    "name": "Placa ‘Goliath’"
   },
   "rct2.mdsaent": {
     "reference-name": "'Medusa' Sign",
-    "name": "Placa 'Medusa'"
+    "name": "Placa ‘Medusa’"
   },
   "rct2.nitroent": {
     "reference-name": "'Nitro' Sign",
-    "name": "Placa 'Nitro'"
+    "name": "Placa ‘Nitro’"
   },
   "rct2.walltxgt": {
     "reference-name": "'Texas Giant' Sign",
-    "name": "Placa 'Texas Gigante'"
+    "name": "Placa ‘Texas Gigante’"
```

```suggestion
    "name": "Placa ‘Texas Giant’"
```

Like “Goliath”, this refers to a sign with the name of the Texas Giant roller coaster.